styleguide/derek/view-templates/stories/stories.ejs

```diff
@@ -4,19 +4,55 @@ var images = [
   'https://752f77aa107738c25d93-f083e9a6295a3f0714fa019ffdca65c3.ssl.cf1.rackcdn.com/logos/underarmor.png',
   'https://752f77aa107738c25d93-f083e9a6295a3f0714fa019ffdca65c3.ssl.cf1.rackcdn.com/logos/yetiLogo_main.png'
 ];
+var quote = 'This is a customer stories fake quote for the featured section at the top of the customer stories homepage. We will probably need to include a character limit so that this does not span more than 3 lines.';
+var featQuotes = [
+  {
+    area: 'Top',
+    logo: 'https://www.rackspace.com/sites/default/files/story-logos/construct-3.png',
+  },
+  {
+    area: 'Center',
+    logo: 'https://www.rackspace.com/sites/default/files/story-logos/sagittarius.png',
+    link: 'https://www.rackspace.com/customer-stories/sagittarius',
+    linkClass: 'story-featured-link'
+  },
+  {
+    area: 'Bottom',
+    logo: 'https://www.rackspace.com/sites/default/files/story-logos/docker_0.png',
+  }
+]
 %>
 
 <!--Header Short-->
-<div class="imacBanner imacBanner-stories">
```

Comment:
So just for my information, are we completely dropping imac banners on the stories page?